src/Raven.Server/Documents/Handlers/DataArchivalHandler.cs

```diff
@@ -0,0 +1,52 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="RevisionsHandler.cs" company="Hibernating Rhinos LTD">
+//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System.Net;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Operations.DataArchival;
+using Raven.Server.Documents.Handlers.Processors.DataArchival;
+using Raven.Server.Routing;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Handlers
+{
+    public class DataArchivalHandler : DatabaseRequestHandler
+    {
+        [RavenAction("/databases/*/data-archival/config", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
+        public async Task GetArchivalConfig()
```

Comment:
Shouldn't this use `DataArchivalHandlerProcessorForGet`?

Reply:
Fixed